lasy/utils/laser_utils.py

```diff
@@ -1385,7 +1385,64 @@ def get_spectral_phase(grid, dim, omega0, method="sum", ordering="zero_center"):
     return phase, omega
 
 
-def get_gdd(grid, dim, omega0, omega_gdd=None, method="sum"):
+def get_gd(grid, dim, omega0, omega_eval=None, method="sum"):
+    r"""
+    Calculate the group delay (GD) of the laser.
+
+    .. math::
+        GD = \frac{\partial \phi(\omega)}{\partial \omega}
+
+
+    Parameters
+    ----------
+    grid : Grid
+        The grid with the field to analyze.
+
+    dim : string
+        Dimensionality of the array. Options are:
+
+        - ``'xyt'``: The laser pulse is represented on a 3D grid:
+                    Cartesian (x,y) transversely, and temporal (t) longitudinally.
+        - ``'rt'`` : The laser pulse is represented on a 2D grid:
+                    Cylindrical (r) transversely, and temporal (t) longitudinally.
+
+    omega0 : float
+        Angular frequency at which the laser envelope is defined.
+
+    omega_eval : float, optional
+        Central angular frequency at which the GDD is calculated, if `None`, `omega0` is used.
+
+    method : string, optional
+        Method of retrieving the phase that is used for calculating the GD. Options are:
+
+        - ``'sum'``: Calculates the spectral phase of the spatially summed field (default).
+        - ``'on-axis'``: Calculates the on-axis spectral phase.
+
+    Returns
+    -------
+    gdd: ndarray of floats (1D)
+        Group delay over the entire spectral range (in s^2)
+
+    gdd0: float
+        Group delay at the center frequency (in s^2)
```

Comment:
Could you check description and units below? 
```suggestion
    gd: ndarray of floats (1D)
        Group delay over the entire spectral range (in s)

    gd0: float
        Group delay at the center frequency (in s)
```